app/services/message_processor.rb

```
@@ -0,0 +1,55 @@
+      else
+        Vote.create(char: char)
+      end
+
+      ProgramChannel.broadcast_to("program_#{program.id}", {
+        action: :message,
+        data: program.reload.view
+      })
+      @working = false
+    end
+    puts consumer
+  end
```

rubocop (1.69.1)

[convention] Assignment Branch Condition size for process is too high. [<8, 59, 7> 59.95/23]

(Metrics/AbcSize)